src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: $card-size;
+  background: $color-main;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  & a {
+    @extend %font-default;
+  }
+
+  & p {
+    @extend %font-default;
+  }
+
+  & h2 {
+    @extend %font-default;
+  }
```

Comment:
it's better not to add styles to tags. better to use specific class names (e.g. `&__name`)